daprdocs/content/en/developing-applications/building-blocks/state-management/state-management-overview.md

```diff
@@ -55,6 +55,15 @@ When strong consistency is used, Dapr waits for all replicas (or designated quor
 
 Read the [API reference]({{< ref state_api.md >}}) to learn how to set consistency options.
 
+### Setting content type
+
+Some state store components might have different ways of maintaining and manipulating data, depending on its type.
+Dapr supports passing content type in [State management API](#state-management-api) as part of request metadata.
```

Comment:
I presume that this is optional. If so, can you say that. And if not supplied what does the contentType default to.